### ISSUE TITLE
Build error: SDL confusion

### ISSUE THREAD

**Issue by holdenlee** [z0w0/helm#99]
Cloning and using cabal install gives the following error:

    src\FRP\Helm.hs:44:18:
    Could not find module `Graphics.UI.SDL'
    It is a member of the hidden package `SDL-0.6.5.1@SDL_26jmephxohM5v84vz7MqXX'.
    Perhaps you need to add `SDL' to the build-depends in your .cabal file.

I'm confused because I thought the new version depends on **sdl2**, not SDL. Nevertheless, I added SDL to build-depends and got another error:

    [ 3 of 13] Compiling FRP.Helm.Engine  ( src\FRP\Helm\Engine.hs, dist\build\FRP\Helm\Engine.o )

    src\FRP\Helm\Engine.hs:7:15:
    Not in scope: type constructor or class `SDL.Window'

    src\FRP\Helm\Engine.hs:8:15:
    Not in scope: type constructor or class `SDL.Renderer'

**Comment by josephleehunsaker** [comment]
I had this same issue. I had to do cabal install for an old version of SDL.

**Comment by z0w0** [comment]
This should be all good now.
